randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -844,8 +848,15 @@ Extra - room_id: [26]
   > Door to Security Access (Upper)
       Space Jump
   > Door to Crumble City
-      # TODO: Look into requirements further and possibly add SWJ trick. The requirement of space jump makes the advanced tricks in crumble city invalid in non-entrance rando.
-      Screw Attack and Space Jump
+      All of the following:
+          Screw Attack
+          Any of the following:
+              Space Jump or Can Single Walljump
+              All of the following:
+                  Level 0 Locks and Speed Booster and Disabled Door Lock Rando and Disabled Entrance Rando
+                  Any of the following:
+                      Shinespark Tricks (Hypermode)
+                      Level 1 Locks and Shinespark Tricks (Hypermode)
```

Comment:
This should be changed. If both shiensaprk tricks are hypermode, this one is never chosen because it has more requirements.
it should isntead be `lv1 + shinespark (expert) + knowledge (advanced)` or something.

Reply:
dunno didnt do the trick. if jagger and you say it isnt that hard, feel free to bump it down to advanced

Reply:
Having them both be hypermode was my mistake, the w/o level 1 was supposed to be expert. but know after thinking about it, I decided they should both be expert.
Don't think this should be advanced tho, ive tried this for several minutes and still cant get it
I also added a wall jump requirement for the upper method instead of knowledge

Reply:
According to jagger, the l0 was considerably more difficult than l1

Reply:
> I also added a wall jump requirement for the upper method instead of knowledge

This isnt a walljump tho. AFAIK you just hold down the jump button

Reply:
the wall jump is in reference to getting enough height for the shine spsark, you can get climb the room in time without it

Reply:
I had been trying to do it before grabbing SA, so I needed to plan around the zoro, but in retrospect, you need SA for the blocks regardless lol. It took longer, but I'm not sure if the one through the bottom door had just been a fluke or if it actually is easier.

Top requires bouncing off the right side to reach the middle platform.

Reply:
Basically you have slightly more time, but the spark height is more precise.

Reply:
@JaggerTSG @herculesbenchpress can one of you at some point get yt vids of you getting both of these sparks, *specifically with screw equipped*? I've been trying my best at it for the past few days and I'm still unable to get it.

Feel free you label all the trick requirements for them however you want, but my personal take is that `Shinespark Tricks` shouldn't be below `Expert`